client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -111,6 +111,10 @@ public final class ApiEndpointsV1
 
     public static final String CONNECTED_CLIENT_STATS_ROUTE = API_V1 + CASSANDRA + "/stats/connected-clients";
 
+    public static final String LIST_JOBS_ROUTE = API_V1 + CASSANDRA + "/jobs";
+    public static final String JOB_STATUS_ROUTE = API_V1 + CASSANDRA + "/jobs/" + JOB_ID_PATH_PARAM + "/status";
+    public static final String NODE_DECOMMISSION_ROUTE = API_V1 + CASSANDRA + "/node/decommission";
```

Comment:
We are using `management` in the prefix, for example /api/v1/cassandra/management/node/decommission for JMX and nodetool operations. IIRC, we discussed this during Sidecar's JMX APIs design meeting

Reply:
Right, and I feel these are different from those (and we arrived at this design during the nodetool APIs review), as they are specific to the administrative node operations like decommission, repair, rebuild etc. We can converge on the URI offline if you'd like.